### PR TITLE
Lab01

### PR DESCRIPTION
**Pull request by bodnarira**

### CODE REVIEW

**Comment by polyakovartem** [comment]
please ask your colleagues to comment on this lab

**Comment by bodnarira** [comment]
> please ask your colleagues to comment on this lab

My colleagues said that everything is right in code

File: lab01.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+def main(x):
+    sum = 0
+    for count in range(x):
+        randnums = random.randrange(1,10)
+        sum += randnums 
+        print(randnums, '; ')
+    print('sum =', sum)
+    print('avg = ', sum/x)
+main(5)
```
Comment (polyakovartem) [inline lab01.py]: https://docs.python.org/3/library/__main__.html

**Review by polyakovartem — CHANGES_REQUESTED** [review]
https://docs.python.org/3/library/__main__.html

**Review by polyakovartem — APPROVED** [review]

**Pull request closed without merging**